test/suites/tuned/microshift-tuned.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+Documentation       Tests for verification on MicroShift's Tuned profile
+
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Test Cases ***
+Current Profile Should Be Hashed
+    [Documentation]    We expect that microshift-tuned keeps a hash of the profile and variables.
+
+    SSHLibrary.File Should Exist    /var/lib/microshift-tuned.yaml
```

Comment:
I'm not sure how the name / docs reflect the contents of the test. Could you explain?

Reply:
Is it better now?